examples/client/cmd/start/start.go

```diff
@@ -26,9 +31,19 @@ func NewCommand() *cobra.Command {
 		Short: "Start the client",
 		Long:  "Start the client",
 		Run: func(cmd *cobra.Command, args []string) {
-			lifecycle.Start(flags.RootCACert, flags.PeerCert,
-				flags.PeerKey, flags.ServerAddr,
-				flags.InsecureSkipVerify)
+			if flags.ApiVersion == "v1alpha1" {
+				lifecycle_v1alpha1.Start(flags.RootCACert, flags.PeerCert,
+					flags.PeerKey, flags.ServerAddr,
+					flags.InsecureSkipVerify)
+			} else if flags.ApiVersion == "v1alpha2" {
```

Comment:
Would it be better to create a separate package for the v1alpha2 implementation?

Reply:
Yes they are two different packages.

Reply:
I think that what Venil is saying is to have two different starts, one for the v1 and the other for v2 (and so on in the future). But parametrizing works for me

Reply:
I think it will be best to keep a single binary and just use arguments for selected versions.